scss/titles.scss

```diff
@@ -0,0 +1,19 @@
+.header-title {
+  padding: 8rem 10rem 3rem;
+}
+
+.title {
+  font-size: $text-big;
+  position: relative;
+  width: 100%;
```

Comment:
Im not sure if you will need to use the whole screen width

Reply:
I think that I don't need it now, but later sure yes.